tests/utils/tensorflow/test_rasa_layers.py

```diff
@@ -0,0 +1,748 @@
+import pytest
+import tensorflow as tf
+
+import numpy as np
+
+from rasa.shared.nlu.constants import TEXT
+from rasa.utils.tensorflow.rasa_layers import (
+    ConcatenateSparseDenseFeatures,
+    RasaFeatureCombiningLayer,
+    RasaSequenceLayer,
+)
+from rasa.utils.tensorflow.constants import (
+    DENSE_INPUT_DROPOUT,
+    SPARSE_INPUT_DROPOUT,
+    DROP_RATE,
+    DENSE_DIMENSION,
+    REGULARIZATION_CONSTANT,
+    CONCAT_DIMENSION,
+    WEIGHT_SPARSITY,
+    DROP_RATE_ATTENTION,
+    KEY_RELATIVE_ATTENTION,
+    VALUE_RELATIVE_ATTENTION,
+    MAX_RELATIVE_POSITION,
+    UNIDIRECTIONAL_ENCODER,
+    HIDDEN_LAYERS_SIZES,
+    NUM_TRANSFORMER_LAYERS,
+    TRANSFORMER_SIZE,
+    NUM_HEADS,
+    SENTENCE,
+    SEQUENCE,
+    MASKED_LM,
+)
+from rasa.utils.tensorflow.exceptions import TFLayerConfigException
+from rasa.utils.tensorflow.model_data import FeatureSignature
+
+
+attribute_name = TEXT
+units_small = 2
+units_bigger = 3
```

Comment:
what small and bigger mean?

Reply:
They literally mean small and bigger, I just needed two constants to create features with different units for my tests 🙂  They could be called `units_1` and `units_2` just as well but I think the current names are clearer.

Reply:
actually I'd prefer `_1` and `_2`, to me `small` and `bigger` means that there is some meaning based on size